ChineseFilter/sections/specific.txt

```diff
@@ -21,6 +21,8 @@ $script,xmlhttprequest,third-party,domain=00ksw.com|01zww.com|0biquge.com|114zw.
 !
 !
 !
+buy.line.me##.mediaTopAd
+buy.line.me##.mediaBottomAd
```

Comment:
All good, but wrong filter. Please move to Japanese.

Reply:
I see. I will move these rules.